examples/clusterscoped/escape-to-host-si-sib-mixed.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: intent.security.nimbus.com/v1
+kind: SecurityIntent
+metadata:
+  name: escape-to-host
+spec:
+  intent:
+    id: escapeToHost
+    description: "A attacker can breach container boundaries and can gain access to the host machine"
+    action: Block
+    params:
+      exclude_resources: ["app:nginx"]
+---
+
+# resources which are having matchLables defined, will not incorporate the exclusion labels
+
+apiVersion: intent.security.nimbus.com/v1
+kind: ClusterSecurityIntentBinding
+metadata:
+  name: escape-to-host-binding
+spec:
+  intents:
+    - name: escape-to-host
+  selector:
+    resources:
+      - kind: Namespace 
+        name: default
+        
+      - kind: Pod
+        namespace: ns3
```

Comment:
Please add name here, and make it omitempty, so that a name/namespace/label selector can be used